README.md

```diff
@@ -50,3 +50,19 @@ To be able to synchronize, you need to set up the publication targets that you w
  - Label: Fill the label with a good name of your choice.
  - Root URL should be the protocol + hostname of the publication target (other TAO platform). Ex: https://demo.taotesting.com/.
  - Auth type: In case of Basic Auth, just use a username and password of a real user in the system that you want to publish. Please make sure that user has proper permissions.
+
+## Publishing of delivery to a remote environment
+
+During extension installation it will:
+- register a new filesystem `publishing`
+- register a new event handler for `DeliveryCreatedEvent`. This event handler will export a QTI test package from which delivery is created and store it in `publishing/{md5_hash_of_delivery_uri}/qti_test_export.zip`
```

Comment:
This event handler will export a QTI test package, create the corresponding delivery and store it in `publishing/{md5_hash_of_delivery_uri}/qti_test_export.zip`

Reply:
It works not exactly as in your sentense. It's triggered when local delivery is created and just exports a QTI test to be able to use it later for remote publishing.